### PR TITLE
07_fukushima

### PR DESCRIPTION
**Pull request by shomiyazaki**

### CODE REVIEW

File: analyses/07_fukushima/07_fukushima_hr_2020_doc.html [hunk 1]
```diff
@@ -401,7 +401,7 @@ <h2>設定</h2>
 </ul></li>
 <li>郡の分割は、現行の区割りにて分割されている場合のみ許容</li>
 <li>離島は、航路で結ばれている本土の市区町村と同一の選挙区にする</li>
-<li>市区町村分割を許容しない案、最大市区町村の分割を許容する案をそれぞれ25,000作成</li>
+<li>逐次モンテカルロ法(SMC)による2つの独立したシミュレーションにより、市区町村分割を許容しない案を20,000作成</li>
```
Comment (kentoyamada100) [inline analyses/07_fukushima/07_fukushima_hr_2020_doc.html:404]: Isn't the number of simulations 2*15,000 ?

File: analyses/07_fukushima/07_fukushima_hr_2020_doc.md [hunk 2]
```diff
@@ -26,7 +26,7 @@
   * 旧市区町村単位の人口（総人口-外国人人口）は、合併後の市区町村の人口（総人口-外国人人口）と旧市区町村単位の日本人人口を元に推計
 * 郡の分割は、現行の区割りにて分割されている場合のみ許容
 * 離島は、航路で結ばれている本土の市区町村と同一の選挙区にする
-* 市区町村分割を許容しない案、最大市区町村の分割を許容する案をそれぞれ25,000作成 
+* 逐次モンテカルロ法(SMC)による2つの独立したシミュレーションにより、市区町村分割を許容しない案を20,000作成
```
Comment (kentoyamada100) [inline analyses/07_fukushima/07_fukushima_hr_2020_doc.md:29]: Isn't the number of simulations 2*15,000 ?

**Review by kentoyamada100 — CHANGES_REQUESTED** [review]

File: analyses/07_fukushima/07_fukushima_hr_2020_doc.md (hunk 2 above)
Comment (shomiyazaki) [reply analyses/07_fukushima/07_fukushima_hr_2020_doc.md]: @kentoyamada100 Thank you for pointing this out. I have pushed the updated version

**Comment by shomiyazaki** [comment]
@kentoyamada100 Thank you for your reviews and comments. I have reflected on what you have pointed out!

**Comment by kentoyamada100** [comment]
@shomiyazaki I've merged this into main. If you haven't already, please upload the relevant data to dataverse

**Pull request merged**